### DISCUSSION TITLE
Discord Server Ban Appeal

### DISCUSSION THREAD

**Discussion by oraclesep** [wormhole-foundation/wormhole#3209] [General] [answered]
Hi,
Please I would like appeal the ban of my account from the Wormhole Discord server. My account was hacked and links were sent randomly to the server before I could do anything about it.

Please kindly pardon me and lift the ban so I can rejoin the server. I realise this may not be the right place to appeal the ban, so I would appreciate if someone can help me.

Thank you so much

Discord ID: oraclex

**Comment by aadam-10** [comment, marked as answer]
hello @oraclesep, the ban was revoked, please try to join again.
